docs/sections/user_guide/cli/drivers/index.rst

```diff
@@ -6,20 +6,20 @@ Drivers
 .. toctree::
    :maxdepth: 1
 
-   cdeps
-   chgres_cube
-   esg_grid
-   filter_topo
-   fv3
-   global_equiv_resol
-   ioda
-   jedi
-   make_hgrid
-   make_solo_mosaic
-   mpas
-   mpas_init
-   orog_gsl
-   sfc_climo_gen
-   shave
-   ungrib
-   upp
+   cdeps/index
+   chgres_cube/index
+   esg_grid/index
+   filter_topo/index
+   fv3/index
+   global_equiv_resol/index
+   ioda/index
+   jedi/index
+   make_hgrid/index
+   make_solo_mosaic/index
+   mpas/index
+   mpas_init/index
+   orog_gsl/index
+   sfc_climo_gen/index
+   shave/index
+   ungrib/index
+   upp/index
```

Comment:
Instead of e.g. a `cdeps.rst` file and a `cdeps/` directory (repeat for each driver), there's now just a `cdeps/` directory, which contains an `index.rst` file. This reduces clutter and unnecessary differences between each driver's set of files.